service/openapi/src/main/java/org/dotwebstack/framework/service/openapi/OpenApiConfiguration.java

```diff
@@ -104,10 +111,25 @@ public OpenApiConfiguration(OpenAPI openApi, GraphQL graphQl, TypeDefinitionRegi
     this.environmentProperties = environmentProperties;
   }
 
+  Optional<RouterFunction<ServerResponse>> staticResourceRouter() {
+    URI staticResourceLocation = ResourceLoaderUtils.getResourceLocation(STATIC_ASSETS_LOCATION);
+
+    if (staticResourceLocation != null) {
```

Comment:
ik zou de geneste `if` vermijden
```suggestion
    if (staticResourceLocation != null && Files.exists(Paths.get(staticResourceLocation))) {
```